cmd/root.go

```diff
@@ -2,21 +2,35 @@ package cmd
 
 import (
 	"github.com/spf13/cobra"
+	"kool-dev/kool/environment"
 )
 
 // CobraRunFN Cobra command run function
 type CobraRunFN func(*cobra.Command, []string)
 
 var version string = "0.0.0-dev"
 
-var rootCmd = &cobra.Command{
-	Use:   "kool",
-	Short: "kool - Kool stuff",
-	Long: `An easy and robust software development environment
+var rootCmd = NewRootCmd(environment.NewEnvStorage())
+
+// NewRootCmd creates the root command
+func NewRootCmd(envStorage environment.EnvStorage) (cmd *cobra.Command) {
+	cmd = &cobra.Command{
+		Use:   "kool",
+		Short: "kool - Kool stuff",
+		Long: `An easy and robust software development environment
 tool helping you from project creation until deployment.
 Complete documentation is available at https://kool.dev/docs`,
-	Version:           version,
-	DisableAutoGenTag: true,
+		Version:           version,
+		DisableAutoGenTag: true,
+		PersistentPreRun: func(cmf *cobra.Command, args []string) {
+			if verbose := cmf.Flags().Lookup("verbose"); verbose != nil && verbose.Value.String() == "true" {
+				envStorage.Set("KOOL_VERBOSE", verbose.Value.String())
+			}
+		},
+	}
+
+	cmd.PersistentFlags().Bool("verbose", false, "execute commands with verbosity")
```

Comment:
One thing, for the sake of language, we are *increasing* verbosity, so maybe it is better something like - `increses output verbosity` for description.